src/openrct2/network/NetworkAction.cpp

```diff
@@ -191,16 +191,9 @@ const std::array<NetworkAction, static_cast<size_t>(NetworkPermission::Count)> N
     NetworkAction{
         STR_ACTION_PARK_PROPERTIES,
         "PERMISSION_PARK_PROPERTIES",
-        {
-            GameCommand::SetParkName,
-            GameCommand::SetParkOpen,
-            GameCommand::SetParkEntranceFee,
-            GameCommand::SetLandOwnership,
-            GameCommand::BuyLandRights,
-            GameCommand::PlaceParkEntrance,
-            GameCommand::RemoveParkEntrance,
-            GameCommand::PlacePeepSpawn,
-        },
+        { GameCommand::SetParkName, GameCommand::SetParkOpen, GameCommand::SetParkEntranceFee, GameCommand::SetLandOwnership,
+          GameCommand::BuyLandRights, GameCommand::PlaceParkEntrance, GameCommand::RemoveParkEntrance,
+          GameCommand::PlacePeepSpawn, GameCommand::ChangeMapSize },
```

Comment:
```suggestion
          GameCommand::PlacePeepSpawn, GameCommand::ChangeMapSize, },
```
This should make clang-format add them on each line again